pkgs/applications/audio/midas/generic.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchurl, lib, libX11, libXext, alsaLib, freetype, brand, type, version, homepage, sha256, ... }:
+stdenv.mkDerivation rec {
+  inherit type;
+  baseName = "${type}-Edit";
```

Comment:
By convention, could you call this attribute `pname`. 
Moreover, the name of a package should not contain any uppercase letters.

Reply:
``type`` is also with uppercase, since it's also used in the url and description.
Can I just do ``pname = "${type}-edit";``,or would you suggest I use a string-manipulation function to change the case?
A function for that probably exists in nix, I've see similar things, but I have no idea how to find it.